pages/index.js

```diff
@@ -15,6 +15,14 @@ export default function Home() {
       </Head>
 
       <main>
+        <nav className="navbar">
+          <h4 className="brand-logo"><img src="./icons/dinosaur.png" width="60"/>Evolution</h4>
+          <ul>
+            <li><a href="#contributor-section">Developers</a></li>
+            <li><a href="#card-section">Explore Next.js</a></li>
+            <li><a href="https://github.com/SamSamskies/evolution" target="_blank">Get Involved!</a></li>
+          </ul>
+        </nav>
```

Comment:
Oh and what do you think about moving this above `main`? I'm thinking move it out into a component called `Header` and then moving it above `main`.

Reply:
I can definitely make the changes in the comments above, I'll push the changes up a bit later today!